src/pybind/mgr/dashboard/frontend/src/app/shared/services/api-interceptor.service.ts

```diff
@@ -36,6 +37,9 @@ export class ApiInterceptorService implements HttpInterceptor {
   intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
     return next.handle(request).pipe(
       catchError((resp: CdHttpErrorResponse) => {
+        if (request.headers.has(interceptorSkipHeader)) {
```

Comment:
It is not necessary to introduce another way to prevent the default error behavior.

Reply:
Thats great :) Thanks